src/SprykerEco/Yves/Payolution/Theme/default/components/molecules/installment/installment.twig

```diff
@@ -0,0 +1,48 @@
+{% define config = {
+    name: 'installment',
+    tag: 'installment',
+} %}
+
+{% block body %}
+    <div class="payolution-form payolution-installment-form payment-form small-12 medium-9 large-6" id="payolutionInstallment">
```

Comment:
Where all these classes are used? `payolution-form payolution-installment-form payment-for`